packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -99,7 +99,7 @@ const logout = async (reason: string): Promise<void> => {
 };
 
 export type AuthContextProviderProps = {
-  user: LoggedUser | AnonymousUser | undefined;
+  user?: LoggedUser | AnonymousUser;
```

Comment:
I personally prefer this, but let me know if people liked the otherside better

Reply:
Same, there's a reason why `?:` exists 😸

Reply:
+1